securing_apps/topics/oidc/java/fuse7/classic-war.adoc

```diff
@@ -83,8 +84,31 @@ To enable the functionality, add this section to your `/WEB_INF/web.xml` file:
 </context-param>
 ----
 
-That component uses `keycloak.config` or `karaf.etc` java properties to search for a base folder to locate the configuration.
-Then inside one of those folders it searches for a file called `<your_web_context>-keycloak.json`.
+The available resolvers for `keycloak.config.resolver` are the following ones:
+
+org.keycloak.adapters.osgi.BundleBasedKeycloakConfigResolver::
+  Default resolver that looks up for the configuration file inside the bundle it
+  is configured from. By default, it loads file named `WEB-INF/keycloak.json` but this file name can be configured via `configLocation` property.
```

Comment:
I am unclear what is being said in "Default resolver that looks up for the configuration file inside the bundle it is configured from."

Are you trying to say this? "This is the default resolver, which requires a configuration file. By default, it loads..."